src/Core/ViewHelper/Traits/CompileWithContentArgumentAndRenderStatic.php

```diff
@@ -118,7 +121,7 @@ protected function buildRenderChildrenClosure()
      */
     public function resolveContentArgumentName()
     {
-        if (empty($this->contentArgumentName)) {
+        if (!property_exists($this, 'contentArgumentName') || empty($this->contentArgumentName)) {
```

Comment:
maybe we should split !property_exists() and empty() here, to avoid writing to a non-existing property below - if the VH does not have the property, we could fall back to a second property that is exclusively used within the trait, or just do the magic each time.
Note accessing a non-existing class property will log a deprecation note with php 8.2

Reply:
Done. Please re-review if it's okay now.